exporter/elasticsearchexporter/model.go

```diff
@@ -998,3 +998,29 @@ func sliceHash(h hash.Hash, s pcommon.Slice) {
 		valueHash(h, s.At(i))
 	}
 }
+
+// convertGeolocationToGeopoint mutates attributes map to merge `geo.location.lat` and `geo.location.lon` to `geo.location`.
+func convertGeolocationToGeopoint(attributes pcommon.Map) {
+	const (
+		lonKey    = "geo.location.lon"
+		latKey    = "geo.location.lat"
```

Comment:
I'm wondering if we should do this for all attributes that end with `geo.location`. It seems likely that this attribute will be embedded in other namespaces, similar to what we've done in ECS.

Then again, in the long run, it would be better to change SemConv, so that we don't need to combine attributes. In that light, it may be better to do the most minimal thing possible.

Reply:
Thanks for the reminder about geo location attributes under namespaces. I've updated the PR.